package.json

```diff
@@ -20,8 +20,8 @@
   "author": "Nomic Labs SRL",
   "license": "MIT",
   "dependencies": {
-    "@resolver-engine/imports-fs": "^0.2.2",
-    "@solidity-parser/parser": "^0.6.0",
+    "@resolver-engine/imports-fs": "0.2.2",
```

Comment:
Is something weird going on with this dependency?

Reply:
No, sorry. There was a typo or something went wrong with npm. Just pushed the typo fix.